code/modules/admin/preferences_verbs.dm

```diff
@@ -1,3 +1,10 @@
+/client/verb/setup_character()
```

Comment:
This is the wrong place for it. This is for admin-only preference verbs.

Reply:
As a ghost you have a ton of procs there as well.
That was my inspiration.

Reply:
Only admin-only verbs are there.